<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFaultsToVoxelGrid</class>
 <widget class="QDialog" name="AddFaultsToVoxelGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListView" name="all2Dmeshes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>215</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="selectedFaults">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>215</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deselectDataButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectDataButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="voxelGridListBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>274</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QLabel" name="voxelGridListLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>   Select voxelgrid:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddFaultsToVoxelGrid</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddFaultsToVoxelGrid</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
